### PR TITLE
Add possibility to hide toggle button in the dxDropDownButton

### PR DESCRIPTION
**Pull request by mrjohn0011**

### CODE REVIEW

File: js/ui/drop_down_button.js [hunk 1]
```diff
@@ -228,13 +235,20 @@ let DropDownButton = Widget.inherit({
     },
 
     _actionButtonConfig() {
```
Comment (DokaRus) [inline js/ui/drop_down_button.js:237]: `get _actionButtonConfig() {` ?

File: js/ui/drop_down_button.js [hunk 2]
```diff
@@ -228,13 +235,20 @@ let DropDownButton = Widget.inherit({
...
+        const showToggleButton = this.option("showToggleButton");
         const defaultConfig = {
             onClick: ({ event }) => {
-                this._actionClickAction({
-                    event: event,
-                    selectedItem: this.option("selectedItem")
-                });
+                if(showToggleButton) {
+                    this._actionClickAction({
```
Comment (DokaRus) [inline js/ui/drop_down_button.js:242]:
```suggestion
                    const selectedItem = this.option("selectedItem);
                    this._actionClickAction({
```

File: js/ui/drop_down_button.js [hunk 3]
```diff
@@ -228,13 +235,20 @@ let DropDownButton = Widget.inherit({
...
+                        event: event,
```
Comment (DokaRus) [inline js/ui/drop_down_button.js]:
```suggestion
                        event,
```

File: js/ui/drop_down_button.js [hunk 4]
```diff
@@ -228,13 +235,20 @@ let DropDownButton = Widget.inherit({
...
+                        selectedItem: this.option("selectedItem")
```
Comment (DokaRus) [inline js/ui/drop_down_button.js:244]:
```suggestion
                        selectedItem
```

File: js/ui/drop_down_button.js [hunk 5]
```diff
@@ -252,17 +266,23 @@ let DropDownButton = Widget.inherit({
         return extend({}, selectedItem, defaultConfig);
     },
 
+    _getButtonGroupItems() {
+        const items = [];
+        items.push(this._actionButtonConfig());
+        if(this.option("showToggleButton")) {
+            items.push({
+                icon: "spindown",
+                width: 24,
```
Comment (DokaRus) [inline js/ui/drop_down_button.js:274]: Is it OK across the all themes?

**Comment by hakimio** [comment]
Isn't it just hiding the "spindown" arrow icon instead of the whole button? Maybe better config name would be "arrowVisible"?

**Comment by mrjohn0011** [comment]
@hakimio, dxDropDownButton is dxButtonGroup. It can contain one or two dxButtons. When we use this widget in single-button mode, the second dxButton must be hidden.
 
However, the user can see it as arrow hiding. Maybe, this option should be renamed. @TikhonovaTatyana, @RomanTsukanov, what do you think about it?

**Review by TikhonovaTatyana — COMMENTED** [review]
I think that TWs should check all names. It's better to create a thread in Teams where all options with their functionality should be listed. Then, please mention me and Roma.

**Review by DokaRus — APPROVED** [review]

**Pull request merged**